app/main/views/suppliers.py

```diff
@@ -130,6 +153,13 @@ def confirm_supplier_details():
     except APIError as e:
         abort(e.status_code)
 
+    frameworks = data_api_client.find_frameworks().get('frameworks')
+    open_framework_slugs = [framework['slug'] for framework in frameworks if framework['status'] == 'open']
+    unconfirmed_open_supplier_frameworks = filter(
```

Comment:
worth making this a helper?

Reply:
Maybe. It's used twice so I get what you mean. Do you think it's worth it?

Reply:
Yeah, i think so. Having the filter and the lambda with two conditionals in the view took me a little while to parse. I think it would be cleaner to shift them out. Also one returns a filter object, the other is cast to a list and I spent a bit of time wondering why.

Reply:
Ah fair enough 👍

Reply:
Maybe.

Maybe they're different enough that it doesn't make sense.